### PR TITLE
sdk: add page

### PR DESCRIPTION
**Pull request by zugzug90**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### The Software Development Kit Manager

SDKMAN! is a tool for managing parallel versions of multiple Software Development Kits on most Unix based systems. It provides a convenient Command Line Interface (CLI) and API for installing, switching, removing and listing Candidates. Formerly known as GVM the Groovy enVironment Manager, it was inspired by the very useful RVM and rbenv tools, used at large by the Ruby community. 

### Java all the way down

Install Software Development Kits for the JVM such as Java, Groovy, Scala, Kotlin and Ceylon. Ant, Gradle, Grails, Maven, SBT, Spark, Spring Boot, Vert.x and many others also supported.

### CODE REVIEW

File: pages/common/sdk.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# sdk
+
+> Tool for managing parallel versions of multiple Software Development Kits.
+> Supports Java, Groovy, Scala, Kotlin, Gradle, Maven, Vert.x and many others.
+
+- Install a specific version of Gradle:
+
+`sdk install gradle {{gradle_version}}`
```
Comment (pxgamer) [inline pages/common/sdk.md]: Is Gradle an example here (i.e. could it be `sdk install kotlin ...`)?

If so, `gradle` should be in token syntax for each example. 👍

File: pages/common/sdk.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`sdk use gradle {{gradle_version}}`
+
+- Check current Gradle version:
+
+`sdk current gradle`
+
+- List all available Software Development Kits:
+
+`sdk list`
```
Comment (pxgamer) [inline pages/common/sdk.md]: Does this list installed SDKs or ones that are available to install? Or both?

File: pages/common/sdk.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Update Gradle to newer version:
```
Comment (sbrl) [inline pages/common/sdk.md]: Perhaps `Update Gradle to the latest version`  would read better?

File: pages/common/sdk.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`sdk upgrade gradle`
+
+- Uninstall particular version of Gradle:
```
Comment (sbrl) [inline pages/common/sdk.md]: Perhaps `Uninstall a particular version`..... would read better?

File: pages/common/sdk.md (hunk 2 above)
Comment (zugzug90) [reply pages/common/sdk.md]: @pxgamer Removed ambiguity

File: pages/common/sdk.md (hunk 4 above)
Comment (zugzug90) [reply pages/common/sdk.md]: Thanks. Sentence has been rephrased.

Comment (zugzug90) [reply pages/common/sdk.md]: Fixed

File: pages/common/sdk.md (hunk 3 above)
Comment (zugzug90) [reply pages/common/sdk.md]: Thanks. Sentence has been rephrased.

File: pages/common/sdk.md (hunk 1 above)
Comment (zugzug90) [reply pages/common/sdk.md]: > Is Gradle an example here 
Correct.

Thanks, commands have been reformatted

**Comment by sbrl** [comment]
Thanks, @zugzug90 :smiley_cat:

**Pull request merged**